features/Dockefile.buildimage

```diff
@@ -0,0 +1,26 @@
+FROM fedora:31
+ENV GOPATH /go
+ENV GOBIN /go/bin
+ENV GOCACHE /go/.cache
+ENV PATH=$PATH:$GOPATH/bin
+ENV GOVERSION golang-1.13.5-1.fc31
+
+# rpms required for building and running test suites
+RUN dnf -y install \
+    $GOVERSION \
+    make \
+    tar \
+    && dnf clean all
+
+# get required golang tools and OC client
+RUN go get github.com/onsi/ginkgo/ginkgo && \
+    go get github.com/onsi/gomega/... && \
+    go get -u golang.org/x/lint/golint && \
+    export latest_oc_client_version=$(curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/ 2>/dev/null | grep -o \"openshift-client-linux-4.*tar.gz\" | tr -d \") && \
+    curl -JL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/${latest_oc_client_version} -o oc.tar.gz && \
+    tar -xzvf oc.tar.gz && \
+    mv oc /usr/bin/oc
```

Comment:
nit: non packaged binaries should go into /usr/local/bin/ (for system wide binaries) or /usr/local/sbin (for root only binaries)

Reply:
Also, removing the oc.tar.gz would be nice:

```
mv oc /usr/local/bin/oc && \
rm -f oc.tar.gz
```

Reply:
Will do, thanks!

Reply:
Done